src/Microsoft.Graph.Core/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Microsoft.Graph.Core",
+  "version": "1.4.0",
+  "description": "Microsoft Graph Core Client Library implements core functionality used by Microsoft Graph Client Libraries.",
+  "copyright": "Copyright (c) Microsoft Corporation",
+  "title": "Microsoft Graph Core Client Library",
+  "authors": [ "$author$" ],
+  "supports": {},
+  "dependencies": {
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
```

Comment:
If possible, remove the dependency on `Microsoft.NETCore.Portable.Compatibility`.  I've seen it cause problems on some platforms (notably Xamarin based ones)